### DISCUSSION TITLE
How would you layout LEDs on the back of Samsung S95b?

### DISCUSSION THREAD

**Discussion by DastardlyDino** [awawa-dev/HyperHDR#768] [General]
Samsung now puts removable covers on the back of some of their TVs. I kind of like them since they help a little bit w/ cable management and cable clean up. But I am a bit stumped. How can I attach an LED strip around the perimeter of my TV while still retaining the ability to remove the covers when needed? There are two covers the HDMI cover on the left side (boxed in red) and the cable channel in the middle (boxed in blue). I would especially like to keep the HDMI cover on the left side accessible after attaching the LED strip(s)
![AxGZPVKJPhpsYJzE6f6v4m](https://github.com/awawa-dev/HyperHDR/assets/47339617/4fc66bc0-35e3-4614-b68d-57bafac8f1e7)
![02CNmXTvTOg5XyM2SpjvQl7-2 v1654171899](https://github.com/awawa-dev/HyperHDR/assets/47339617/384f0790-0cec-4ac4-b0a7-64b45683586a)
![Samsung-QE65S95B-stand-uncovered-1536x1024](https://github.com/awawa-dev/HyperHDR/assets/47339617/51481cab-7f16-40b3-9674-e6652671ecca)
![XgMJHfPD36LRMcYNrGQ6pk-1200-80](https://github.com/awawa-dev/HyperHDR/assets/47339617/2a5fd253-c9e6-4632-9306-0475f5499079)
.

**Comment by thecourteener** [comment]
No help but I have the same TV, mines wall mounted so doesn't have the stand attached. I left the hdmi cover off because you can't see it at all on the wall and ran the strip straight along the bottom

**Reply by DastardlyDino** [reply]
Thanks for the reply still. Good to know I am not talking into the void. Someone on Reddit recommended using JST connectors which I think is a pretty good idea.

Unrelated but what are you using to capture content?

**Comment by wilhelmthorpe** [comment]
I would build a small wooden frame for your LEDs. That way you can have them at all the same distance from the wall, and adjust the distance as needed.
